pkg/commands/archive.go

```diff
@@ -15,6 +17,13 @@ import (
 	"github.com/Azure/acr-builder/pkg/constants"
 )
 
+var maxHeaderSize = 512
```

Comment:
>512 [](start = 20, length = 3)

I see the max is 6 bytes from the following list

Reply:
Because of below discussion I have scope this down to 4 bytes